FEXCore/Scripts/config_generator.py

```diff
@@ -217,6 +217,20 @@ def print_man_environment_tail():
     ],
     "''", True)
 
+    print_man_env_option(
+    "FEX_PORTABLE",
+    [
+    "Forces FEX in to a more portable installation",
+    "This does a couple of things:",
+    "- Skips executing binfmt_misc handlers through execve",
+    "- global config (typically installed in /usr/share/fex-emu/) disabled",
+    "- FEX_APP_CONFIG_LOCATION defaults to FEXInterpreter relative paths instead of {$HOME, $XDG_DATA_HOME}/.fex-emu/",
+    "  - <FEXInterpreterPath>/fex-emu/"
+    "- FEX_APP_DATA_LOCATION defaults to FEXInterpreter relative path instead of {$HOME, $XDG_DATA_HOME}/.fex-emu/",
+    "  - <FEXInterpreterPath>/fex-emu/"
```

Comment:
What does "more portable" mean and how would the user know the limitations? Wouldn't it be more sensible to just call it "portable" and instead outline possible *exceptions*?

Something like this might be a starting point for improvement, though it doesn't list the limitations since I don't know what they are:
```suggestion
Allows FEX to run without installation. Global locations for configuration and binfmt_misc are ignored. These files are instead read from <FEXInterpreterPath>/fex-emu/ by default.

For further customization, the environment variables FEX_APP_CONFIG_LOCATION and FEX_APP_DATA_LOCATION can be used.
```

Reply:
Updated text